src/script/common/c_converter.cpp

```diff
@@ -540,9 +540,9 @@ v3s16 getv3s16field_default(lua_State *L, int table,
 }
 
 void setstringfield(lua_State *L, int table,
-		const char *fieldname, const char *value)
+		const char *fieldname, const std::string &value)
```

Comment:
does this not introduce some/many(?) unneeded conversions to std::string and back?
how often is setstringfield used with `char*`?

Reply:
It's used exactly 1 times: `l_http.cpp`